docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -44,24 +44,15 @@ cat <<EOF | kubectl apply -f -
 apiVersion: serverless.kyma-project.io/v1alpha2
 kind: Function
 metadata:
-  labels:
-    serverless.kyma-project.io/build-resources-preset: local-dev
-    serverless.kyma-project.io/function-resources-preset: S
-    serverless.kyma-project.io/replicas-preset: S
   name: lastorder
   namespace: default
 spec:
   replicas: 1
   resourceConfiguration:
     function:
       profile: XS
```

Comment:
```suggestion
      profile: S
```
This will be equal with former version `serverless.kyma-project.io/function-resources-preset: S`